lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -210,5 +210,230 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_CHARGEITEM_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': 'e2a7275f-fe18-4885-a39b-8d798137693f',
+      'type': 'searchset',
+      'total': 1,
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr.devcerner.com/r4/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/ChargeItem?_id=292870306'
```

Comment:
We want to update this to a fhir-open.cerner.com call I believe. We might be able to just use the read and update the links and such manually.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/653/commits/a70116500ae99819757d9ec2d247974c82985b96#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR222-R227